packages/url/src/test/index.test.js

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
> The /** inthe above comment is on is for a file header comment @gziolo, not Jest 😏

I put this comment in random place, sorry for confusion. We use those comments for dependencies in Calypso. I personally hope we don't inherit them to WordPress world :D